clusterloader2/pkg/provider/aws.go

```diff
@@ -61,3 +62,8 @@ func (p *AWSProvider) RunSSHCommand(cmd, host string) (string, string, int, erro
 	user := defaultSSHUser()
 	return sshutil.RunSSHCommand(cmd, user, host, signer)
 }
+
+// Metadata returns provider-specific test run metadata.
```

Comment:
These comments don't bring any value IMHO. I'd leave the comment in the interface but remove them from implementations.